shared/tasks/browser.mjs

```diff
@@ -33,18 +28,14 @@ export async function screenshots() {
 
   // Screenshot components
   for (const componentName of componentNames) {
-    const componentExamples = await getExamples(componentName)
-
-    // Screenshot "default" example
-    await screenshotComponent(browser, componentName)
-
-    // Screenshot any other examples with 'screenshot: true'
-    const otherExamples = Object.keys(componentExamples).filter(
-      (key) => componentExamples[key].fixture.screenshot
+    const allExamples = await getExamples(componentName)
+    const componentExampleNames = Object.keys(allExamples).filter(
```

Comment:
With the `for`'s on line 30 and 36 and this `filter` we're looping 3 times. What do you think about moving the third loop here instead? So we'd get all the examples for each component, loop through them, check that `allExamples[exampleName].fixture.screenshot` exists and if so, run `screenshotComponent`.

This way we could also reuse `fixture.screenshot`.

Reply:
Good points, both. I've added a bit to the JSDoc on the variants, and I've swapped out `.filter` for a `.forEach` and doing the screenshots within that.